src/components/IconButtonBar/_mixins.scss

```diff
@@ -36,13 +36,12 @@
         width: $icon-button-bar__sizing__dimensions;
       }
     }
-  }
-
-  &__overflow-menu-options {
-    box-shadow: unset;
 
-    &.#{$prefix}--overflow-menu-options::after {
-      height: 0;
```

Comment:
This shouldn't be hidden or zeroed out, as Carbon appears to use it as a way to prevent the menu's `box-shadow` from covering part of the overflow menu button.